site/src/sphinx/known-issues.rst

```diff
@@ -17,3 +19,11 @@ Known issues
   - Consider enforcing network-level access control over Thrift calls.
   - Note that the Thrift RPC layer is left only for backward compatibility, and will be removed in the future,
     in favor of the REST API.
+
+Previously known issues
+=======================
+- A Reflected Cross-site Scripting (XSS) vulnerability
+
+  - A Search feature of Central Dogma allows remote attacker to perform unauthorized actions.
+    This issue affects: Central Dogma 0.17.0 version 0.40.1 and prior versions.
+    Please check `CVE-2009-6002`_ to get more information.
```

Comment:
Because the link will be used only once, you could just do an inline reference:

    `CVE-2009-6002 <https:....>`_

Reply:
Thanks!

Reply:
I think there is a typo. 

`CVE-2009-`  -> `CVE-2019`

Reply:
fixed